app_demo.rb

```diff
@@ -0,0 +1,103 @@
+require './lib/order'
+require './lib/submitorder'
+
+class Takeaway
+
+  attr_accessor :menu1, :order1, :name, :price, :menu_item1, :total, :to_mobile, :from_mobile, :message_body, :message, :account_sid, :auth_token, :submitorder1, :quantity
+
+  def initialize
+    @menu1 = Menu.new Dish
+  end
+
+  def add_dish
+    while true
```

Comment:
Style/InfiniteLoop: Use `Kernel#loop` for infinite loops. (https://github.com/bbatsov/ruby-style-guide#infinite-loop)